easybuild/easyconfigs/t/Tk/Tk-8.6.8-iomkl-2018a-no-X11.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'Tk'
+version = '8.6.8'
+versionsuffix = '-no-X11'
```

Comment:
@branfosj Why is this easyconfig included in here? It's not required for the dependencies, and the `Tk-8.6.8-iomkl-2018a.eb` we already have should be preferred anyway...

Can you please remove this easyconfig file from your PR?